src/KubeLibrary/KubeLibrary.py

```diff
@@ -304,6 +304,14 @@ def filter_pods_containers_by_name(self, pods, name_pattern):
                     containers.append(container)
         return containers
 
+    def filter_service_names(self, service):
+        """Filter service  names for list of service .
+        Returns list of strings.
+        - ``service``:
```

Comment:
Small thing here, but should be plural - 'services'

Reply:
Done.